Index: start.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>PORT=400 npm run start
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/start.sh b/start.sh
--- a/start.sh	(revision aa2c703c611cdd29cd585a2b29a1f813dec29bb9)
+++ b/start.sh	(date 1727291766742)
@@ -1,1 +1,1 @@
-PORT=400 npm run start
\ No newline at end of file
+PORT=4000 npm run start
\ No newline at end of file
